packages/vue/_src/hooks.ts

```diff
@@ -206,58 +206,67 @@ export function make<R, E, A>(self: Effect<R, E, FetchResponse<A>>) {
   return tuple(result, latestSuccess, execute)
 }
 
+export type MutationResult<E, A, I = never> = readonly [
+  {
+    loading: Ref<boolean>
+    success: Ref<A | undefined>
+    error: Ref<E | undefined>
+  },
```

Comment:
This is the one I would've put in an interface. the tuple imo just overcomplicates